src/elements/content-sidebar/Sidebar.js

```diff
@@ -61,8 +61,8 @@ type Props = {
     metadataEditors?: Array<MetadataEditor>,
     metadataSidebarProps: MetadataSidebarProps,
     onAnnotationSelect?: Function,
-    onOpenChange?: (isOpen: boolean) => void,
-    onPanelChange?: (name: string) => void,
+    onOpenChange?: (isOpen: boolean, isInitialState?: boolean) => void,
```

Comment:
Would it be more intuitive to make isInitialState non optional? This seems to be an important part of this functionality and we're essentially treating not passing the optional parameter back as the false value anyway.

Reply:
Hmm, I didn't want to change the previous implementation, hence making the second parameter optional, but I think you're right, I'll change this, so that every use is intentional passing either `false` or `true`.

Reply:
Done